website/docs/reference/resource-configs/grants.md

```diff
@@ -22,7 +22,22 @@ dbt encourages you to use grants as resource configs whenever possible in Core v
  
 For more information on hooks, see [Hooks & operations](/building-a-dbt-project/hooks-operations).
 
-You can set grants in `dbt_project.yml` and as a `config` yaml property that applies to the entire dbt project.
+## Definition
+
+You can use the `grants` field to set permissions or grants for a resource. When you run a model, seed or seed, or snapshot a snapshot, dbt will run `grant` and/or `revoke` statements to ensure that the permissions on the database object match the `grants` you have configured on the resource.
+
+Like all configurations, `grants` will be included in dbt project metadata, including [the manifest artifact](dbt-artifacts/manifest-json).
```

Comment:
will these be surfaced in dbt docs page for a given model now as well?

Reply:
Not yet - but they could be! I think it would take just a bit of `dbt-docs` code to make that happen.